views/index.html

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+    <link href="/static/main.css" rel="stylesheet">
+    <title>Service Unavailable</title>
```

Comment:
Should the title have "Transfer Digital Records" in it? Not sure.

We were supposed to review the titles as part of the content review, but I don't think we did. I'll check and add a card.

Reply:
Ok. I guess leave it like this now? It's a quick change later if we want to do it.